src/pyqttoast/toast.py

```diff
@@ -480,6 +481,15 @@ def __calculate_position(self):
             y = (current_screen.geometry().y()
                  + Toast.__offset_y + y_offset)
 
+        elif Toast.__position == ToastPosition.CENTER:
+            x = (current_screen.geometry().x()
+                 + current_screen.geometry().width() / 2
+                 - self.__notification.width() / 2)
+            y = (current_screen.geometry().y()
+                 + current_screen.geometry().height() / 2
+                 - self.__notification.height() / 2
+                 + Toast.__offset_y + y_offset)
```

Comment:
Looks good but I was thinking that maybe the `+ Toast.__offset_y` should be removed here since with the other positions, the offset is ignored for every axis the toast is centered on (e.g. the x-axis for BOTTOM_MIDDLE) so when centering on both the x and y-axis, it should be completely centered. Let me know what you think :)